scripts/tags/exturl.js

```diff
@@ -4,8 +4,11 @@
 
 'use strict';
 
-var util = require('hexo-util');
+/*jshint camelcase: false */
+var util = require(hexo.base_dir + 'node_modules/hexo/node_modules/hexo-util');
```

Comment:
According to the [algorithm of installing a package with npm](https://docs.npmjs.com/cli/install#algorithm), `hexo-util` should be installed in the site's `node_modules` instead of Hexo's. In this case, it should be OK to use `require('hexo-util')` in tag scripts directly.